.github/ISSUE_TEMPLATE/bug_report_prompts.yml

```diff
@@ -0,0 +1,76 @@
+name: Bug report for App Permissions prompts
+description: Issues with prompts and specific apps? Let us know.
+title: "[Prompting bug]: "
+labels:
+  - bug
```

Comment:
It might be worth adding a dedicated label for prompting bugs, so we can filter them easily in the issue tracker here

Reply:
I think maybe we want them for the different features of the security center and then add multiple. So,
  - `prompting`, `bug`
  - `prompting`, `feature-request`
  - etc

Rather than creating `prompting-bug` and `prompting-feature-request` as specific labels

Reply:
Just tweaked it, using `enhancement` for feature requests as that label is [standard](https://github.com/canonical/desktop-security-center/labels)